### PR TITLE
[google-one-tap] Add log_level option

### PR DESCRIPTION
**Pull request by jameshfisher**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [here](https://stackoverflow.com/a/72883451/229792)

### CODE REVIEW

**Review by MysteryBlokHed — COMMENTED** [review]
If you update the tests (`test/google-one-tap-global.test.ts`) to include the new `log_level` property, the PR can be merged without a maintainer's approval.
For example, you could add this after line 16:

```typescript
    log_level: 'debug'
```

**Review by MysteryBlokHed — APPROVED** [review]
LGTM!

**Comment by jameshfisher** [comment]
Ready to merge

**Pull request merged**
